packages/rdma-core/rdma-core.spec

```diff
@@ -0,0 +1,165 @@
+Name: %{_cross_os}rdma-core
+Version: 53.1
+Release: 1%{?dist}
+Summary: RDMA core userspace libraries and daemons
+License: MIT OR GPL-2.0-only
+Source0: https://github.com/linux-rdma/rdma-core/releases/download/v%{version}/rdma-core-%{version}.tar.gz
+Source100: rdma-core-tmpfiles.conf
+
+BuildRequires: cmake >= 2.8.11
+BuildRequires: %{_cross_os}libnl-devel
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}libnl
+
+%description
+%{summary}.
+
+%package devel
+Summary: RDMA core development libraries and headers
+# For now we are only providing the libibverbs-devel as it
+# will be needed to build EFA Device Plugin.
+Provides: %{_cross_os}libibverbs-devel
```

Comment:
This was mimicing the behavior in the original [rdma-core spec](https://github.com/linux-rdma/rdma-core/blob/master/suse/rdma-core.spec#L171) so that one trying to install `libibverbs-devel` would get `rdma-core-devel`. 

Today if you try to install the libibverbs-devel package on fedora, it will try to install rdma-core-devel 
```
~> sudo dnf install libibverbs-devel
Last metadata expiration check: 1:02:28 ago on Mon 28 Oct 2024 07:55:50 PM UTC.
Dependencies resolved.
===================================================================================================================================================================================================
 Package                                              Architecture                               Version                                          Repository                                  Size
===================================================================================================================================================================================================
Installing:
 rdma-core-devel                                      x86_64                                     48.0-4.fc40                                      fedora                                     429 k
Installing dependencies:
 infiniband-diags                                     x86_64                                     48.0-4.fc40                                      fedora                                     328 k
 libibumad                                            x86_64                                     48.0-4.fc40                                      fedora                                      26 k
 libibverbs                                           x86_64                                     48.0-4.fc40                                      fedora                                     432 k
 librdmacm                                            x86_64                                     48.0-4.fc40                                      fedora                                      72 k
```

Let me know what you think. I can also try updating the spec file to explicitly group the dedicated `libibverbs` and `libibverbs-devel` if that is what we want.

Reply:
> Let me know what you think. I can also try updating the spec file to explicitly group the dedicated `libibverbs` and `libibverbs-devel` if that is what we want.

Often these provides are added for backwards compatibility. We don't need to bother with that since we haven't packaged this software before. We can just have `rdma-core` and `rdma-core-devel`.

Reply:
Got it, removed the `libibverbs-devel` and only kept `rdma-core` and `rdma-core-devel`.